post-scan-actions/aws-python-promote-or-quarantine/terraform/variables.tf

```
@@ -0,0 +1,63 @@
+# Configure the variables here
+variable "region" {
+  type = string
+  description = "AWS region"
+  default = "us-east-1"
+  validation {
+    condition     = can(regex("[a-z][a-z]-[a-z]+-[1-2]", var.region))
+    error_message = "Must be valid AWS Region names."
+  }
+ }
+
+variable "ScanningBucketName" {
+  type = string
+  default = ""
+  description = "The S3 bucket scanned by Trend Micro Cloud One File Storage Security."
+ }
+
+variable "ScanResultTopicARN" {
+  type = string
+  default = ""
+  description = "The ARN of the scan result SNS topic in storage stack."
+ }
+
+variable "PromoteBucketName" {
+  type = string
+  default = ""
+  description = "[Optional] The bucket name used to promote files without malicious intent. Leave the bucket name blank to disable promoting."
+ }
+
+variable "PromoteMode" {
```

Please follow the naming convention like we did before in GCP plugin's Terraform template for consistency. Apply the convention to the other variables too.
https://github.com/trendmicro/cloudone-filestorage-plugins/blob/0c1e219478b3e8c2ff6aaf762a65847c0c3a9457/post-scan-actions/gcp-python-promote-or-quarantine/variables.tf#L48
```suggestion
variable "promote_mode" {
```